lualib/rspamadm/ratelimit.lua

```diff
@@ -0,0 +1,202 @@
+local argparse = require 'argparse'
+local redis = require 'lua_redis'
+local logger = require 'rspamd_logger'
+
+local parser = argparse()
+        :name "ratelimit"
+        :description "Manage ratelimit functional"
+        :help_description_margin(32)
+        :command_target('command')
+        :require_command(true)
+
+local track_limits = parser:command 'track_limits'
+                           :description 'Track last limits of ratelimit module'
+
+track_limits:option "-q --quantity"
+            :description("Number of limits to track")
+            :argname("<quantity>")
+            :default(1)
+
+
+local upgrade_bucket = parser:command 'upgrade_bucket'
+                             :description 'Upgrade certain bucket'
+
+upgrade_bucket:argument "prefix"
+              :description("Prefix of bucket to operate with")
+              :args(1)
+upgrade_bucket:option "-b --burst"
+              :description("Burst to set")
+              :argname("<burst>")
+              :args("?")
+upgrade_bucket:option "-r --rate"
+              :description("Rate to set")
+              :argname("<rate>")
+              :args("?")
+upgrade_bucket:option "-s --symbol"
+              :description("Symbol to set")
+              :argname("<symbol>")
+              :args("?")
+upgrade_bucket:option "-B --dynamic_burst"
+              :description("Dynamic burst to set")
+              :argname("<dynb>")
+              :args("?")
+upgrade_bucket:option "-R --dynamic_rate"
+              :description("Dynamic rate to set")
+              :argname("<dynr>")
+              :args("?")
+
+local unblock_bucket = parser:command 'unblock_bucket'
+                             :description 'Unblock certain bucket'
+
+unblock_bucket:argument "prefix"
+              :description("Prefix of bucket to operate with")
+              :args(1)
+
+local unblock_buckets = parser:command 'unblock_buckets'
```

Comment:
Why can't it be an option for `unblock`?

Reply:
He can get it from the first command. So i created another command to unblock plenty of buckets in a time. And, of course, it could be merged with this command, but I thought it would not be very user friendly to use the same command for multiple update and single update(that uses specific prefix) in one command